README.md

```diff
@@ -1,9 +1,37 @@
-# Bootcamp-Project-1
+# [Traileremma](https://willberner.github.io/Trailr/)
 
 A basic web application for searching, filtering, and sorting movies. Created with data pulled from the OMDb API and styled with Materialize.
+A Front-End web application for searching for movies. Users can search a movie by title, select which movie they're interested in, and view information about the movie such as its IMDB critic rating, trailer, runtime, if the movie is on Netflix to stream, and other related material.
 
-Created for UNC-CH's 2021 Full Stack Coding Bootcamp by Will Berner, Muiasar Al Ani, Casey Twine, and Zach Holloway.
+## Instructions for use
+1. Use the search card on the homescreen to search for a movie title you think you might be interested in.
 
-A live link to the website can be found here:
+2. The resulting page will display results from your search, and if you see the movie you were looking for you can click on it to be brought to a third page.
 
-https://willberner.github.io/Bootcamp-Project-1/
+3. Check out the third page to watch the movie's trailer, or to view other detailed information!
+
+## Built With
+* HTML
+* CSS
+* Javascript
+* Materialize CSS Library
+* Three APIs
+  * Open Movie Database (OMDb) API
+  * Google Youtube Data API v3
+  * Streaming-Availability API via RapidAPI
+
+## Demo
```

Comment:
I figure after we have everything completely in place I'll replace this with a short video or gif of searching a movie.